backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -1221,13 +1228,15 @@ internal class FunctionGenerationContext(val function: LLVMValueRef,
     internal fun epilogue() {
         appendingTo(prologueBb) {
             if (needsRuntimeInit) {
+                assert(!forbidRuntime)
```

Comment:
Please use `require` or `check` instead of `assert`. Assertions can be disabled and somewhat don't match the semantics required here (~precondition).

Reply:
(same for the other new assertions)

Reply:
Right, I forgot those existed, thanks!